### ISSUE TITLE
Missing py.typed file for mypy_django_plugin

### ISSUE THREAD

**Issue by GergelyKalmar** [typeddjango/django-stubs#764]
# Bug report

## What's wrong

When importing `mypy_django_plugin` I'm getting the following error with mypy:

```
1:1: error: Skipping analyzing "mypy_django_plugin": found module but no type hints or library stubs  [import]
1:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

## How is that should be
There should be no errors.

## System information

- OS: Ubuntu 20.04 LTS
- `python` version: 3.8.10
- `django` version: 3.2.9
- `mypy` version: 0.910
- `django-stubs` version: 1.9.0
- `django-stubs-ext` version: 0.3.1

**Comment by sobolevn** [comment]
Can you please tell me more about your usecase?

Our mypy plugin is not usually type-checked by itself.

**Comment by GergelyKalmar** [comment]
Sure! I'm patching the plugin to use the inicfg when looking for the Django settings module path (instead of having it read it from pyproject.toml):

```python
from mypy_django_plugin import main as mypy_django_plugin_main

def pytest_configure(config: Any) -> None:
    def django_settings_module(*_args: Any, **_kwargs: Any) -> str:
        return config.inicfg['DJANGO_SETTINGS_MODULE']

    mypy_django_plugin_main.extract_django_settings_module = django_settings_module
```

The broader use case is that I've found it silly that I need to specify the same thing in multiple different `pyproject.toml` sections when working with various Django plugins (https://github.com/PyCQA/pylint-django takes it from the `[tool.pylint.MASTER]` section, https://github.com/pytest-dev/pytest-django uses `[tool.pytest.ini_options]`, in django-stubs it is taken from the `[tool.django-stubs]` section), so I wrote a plugin that injects the pytest.ini_options value into the other plugins. I wish there was an easier way to do this without an ugly patch, but anyhow, mypy is complaining that the mypy plugin is not type checked. I think the plugin has type hints already so we only need to add the `py.typed` file there to make it importable too.
